dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/FileFormatHelper.java

```diff
@@ -66,4 +75,18 @@ public static void updateFileFormats(Set<? extends Version> versions, final File
         });
         return fileFormatsFromDB;
     }
+
+    public static String calcSHA1(String content) {
```

Comment:
Suggest you return an  `Optional<String>` since you can return a null. Your code where you use this function doesn't check for a null, which makes an argument for why you should be returning an Optional. :)

On the other hand, if you think this is such an unlikely/impossible occurrence, which is possible (if UTF-8 encoding is not present, nothing's going to work, probably the same for SHA-1 algorithm not being present, not sure about UnsupportedOperationException), then maybe you should return a String and throw an exception.

Reply:
returning optional